rcsbsearchapi/search.py

```diff
@@ -347,6 +351,55 @@ def __init__(self, value: str, pattern_type: Optional[SeqMode] = "simple", seque
                                                                       "sequence_type": sequence_type})
 
 
+class StructSimilarityQuery(Terminal):
+    """Special case of a terminal for structure similarity queries"""
+
+    def __init__(self, structure_search_type: StructSimEntryType = "entry_id",
+                 value: Optional[str] = None,
+                 input_structure_type: Optional[StructSimInputType] = "assembly_id",
+                 input_structure_id: str = "1",
+                 operator: StructSimOperator = "strict_shape_match",
+                 target_search_space: StructSimSearchSpace = "assembly"
+                 ):
+        if structure_search_type == "entry_id":
+            if input_structure_type == "assembly_id":
+                super().__init__(service=STRUCT_SIM_SEARCH_SERVICE, params={
+                    "operator": operator,
+                    "target_search_space": target_search_space,
+                    "value": {
+                        "entry_id": value,
+                        "assembly_id": input_structure_id
+                    }
+                })
+            elif input_structure_type == "chain_id":
+                super().__init__(service=STRUCT_SIM_SEARCH_SERVICE, params={
+                    "operator": operator,
+                    "target_search_space": target_search_space,
+                    "value": {
+                        "entry_id": value,
+                        "asym_id": input_structure_id
+                    }
+                })
+        elif structure_search_type == "file_url":
+            super().__init__(service=STRUCT_SIM_SEARCH_SERVICE, params={
+                "operator": operator,
+                "target_search_space": target_search_space,
+                "value": {
+                    "url": value,
+                    "format": input_structure_id
```

Comment:
The variable `input_structure_id` is confusing here. Can it be something more generic that is applicable to search by `entry_id` or `file_url`?